edith-step/src/UserChart.js

```diff
@@ -163,8 +164,22 @@ const PieChart = () => {
         console.log(response);
         let itemNames = [];
         let itemValues = [];  
-        const items = JSON.parse(response.data.items);
-        console.log(items);
+        let items = [];
+        const itemsList = JSON.parse(response.data.items);
+        if (props.dateSelection != "") {
```

Comment:
Is this equivalent to..
` if (props.dateSelection.length > 0`
OR 
`if (!props.dateSelection) {`.

^  as these might be cleaner

I do believe the first is considered faster due to interpreter advantages. See https://stackoverflow.com/questions/154059/how-can-i-check-for-an-empty-undefined-null-string-in-javascript

Reply:
I changed it to the first one.